From f761d4451823a043cbe3b3af8a72e231c8ae1386 Mon Sep 17 00:00:00 2001
From: Thefrank <1910378+Thefrank@users.noreply.github.com>
Date: Sat, 22 Oct 2022 13:59:53 -0700
Subject: [PATCH] freebsd has community supported asp.net runtime

Since 3.1 aspnetcore has been buildable under FreeBSD.
Since 6.0 aspnetcore has an RID for FreeBSD-x64
---
 src/redist/targets/SetBuildDefaults.targets | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/src/redist/targets/SetBuildDefaults.targets b/src/redist/targets/SetBuildDefaults.targets
index eac457403bd..f01c618b167 100644
--- a/src/redist/targets/SetBuildDefaults.targets
+++ b/src/redist/targets/SetBuildDefaults.targets
@@ -20,9 +20,6 @@
         AND '$(Rid)' != 'ubuntu.18.04-x64'">true</IncludeSharedFrameworksForBackwardsCompatibilityTests>-->
       <HighEntropyVA>true</HighEntropyVA>
 
-      <!-- There is no asp.net runtime available for freebsd at the moment. Once there is one, we can remove the line below -->
-      <IncludeAspNetCoreRuntime Condition=" '$(IncludeAspNetCoreRuntime)' == '' AND $(Rid.StartsWith('freebsd'))">false</IncludeAspNetCoreRuntime>
-
       <!-- Only use asset target fallback that we set (not implicit one to net461). -->
       <!--<DisableImplicitAssetTargetFallback>true</DisableImplicitAssetTargetFallback>-->
 
